ddp-workspace/projects/ddp-testboston/src/app/components/footer/footer.component.html

```diff
@@ -55,7 +55,8 @@
                             routerLinkActive="footer-link_active" translate>Navigation.Invitation</a>
                     </li>
                     <li>
-                        <a class="Link footer-link" href="kit-instructions.pdf" target="_blank" translate>
+                        <a class="Link footer-link" [href]="language + '/kit-instructions.pdf'" target="_blank"
```

Comment:
Should we internationalize PDFs' names?

Reply:
I think it's fine -- user can save it with whatever name they want.